Multiparty/README.md

```diff
@@ -0,0 +1,295 @@
+# Multiparty sample
+
+This sample application shows how to arrange videos and to toggle the mic and camera in multiparty calls using OpenTok React Native SDK.
+
+The app builds upon the Basic Video Chat sample app.
+
+## Setup
+
+1. Install the required node modules: `npm install`
+
+2. For iOS, install the Podfile's dependencies: `cd ios/ && pod install`
+
+3. In the App.js file, set the `apiKey`, `sessionId`, and `token` properties to your Vonage Video API key (project ID), a Vonage Video session ID, and a token for that session.
+
+For testing, you can use the [OpenTok playground](https://tokbox.com/developer/tools/playground/) to create sessions, publish streams to the session from a web client, and view the stream published from the client using the OpenTok React Native SDK. Open the playground in multiple tabs to publish multiple streams to the session (from the browser). And mute the computer's speaker to prevent audio feedback
```

Comment:
```suggestion
For testing, you can use the [OpenTok playground](https://tokbox.com/developer/tools/playground/) to create sessions, publish streams to the session from a web client, and view the stream published from the client using the OpenTok React Native SDK. Open the playground in multiple tabs to publish multiple streams to the session (from the browser). And mute the computer's speaker to prevent audio feedback.
```